src/main/java/lotto/view/OutputView.java

```diff
@@ -0,0 +1,44 @@
+package lotto.view;
+
+import java.util.List;
+import java.util.Map;
+import lotto.domain.Lotto;
+import lotto.domain.Rank;
+
+public class OutputView {
+
+
+    public static final String ATTENDANCE_PURCHASE_MESSAGE = "%d개를 구매했습니다.\n";
+    public static final String INFO_ATTENDANCE_STATISTICS_MESSAGE = "당첨 통계\n---";
+    public static final String ATTENDANCE_STATISTICS_MESSAGE = "%s - %d개\n";
+    public static final String ATTENDANCE_REVENUE_MESSAGE = "총 수익률은 %.1f%%입니다.";
+
+
+    private OutputView() {
+    }
+
+    public static OutputView create() {
+        return new OutputView();
+    }
+
+    public void printPurchaseLottos(List<Lotto> lottos) {
+        System.out.printf(ATTENDANCE_PURCHASE_MESSAGE, lottos.size());
+        for (Lotto lotto : lottos) {
+            System.out.println(lotto.toString());
+        }
```

Comment:
해당 `for`문을 메소드 참조를 활용하면 간결하게 표현할 수 있어요. 스트림 API를 활용해보시는 것도 추천드립니다!
```suggestion
        lottos.forEach(System.out::println);
```

Reply:
`toString()`메소드가 있어서 안될 거라 생각했었는데 가능했군요..! 알려주셔서 감사합니다 👍